api/Onboard.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import (
+	"fmt"
+	"github.com/operate-first/opfcli/models"
+	"strings"
+)
+
+func (api *API) Onboard(path string, projectDisplayName string, disableLimitrange bool) error {
+	onboardRequest, err := models.OnboardRequestFromYAMLPath(path)
+
+	if err := api.CreateGroup(
```

Comment:
Shouldn't this be calling `api.CreateProject`, which:

- Creates the namespace
  - Including quotas
  - And limitranges
- Creates the group
- Creates the role binding

Or have I misunderstood what this command is supposed to do?

Reply:
So I think that onboarding can have multiple namespaces with a single group and role binding but projects only have 1 namespace. That's why I created the group and rolebinding first before looping over the namespaces.